web/src/pages/Configuration/Moderation/index.tsx

```diff
@@ -112,29 +122,34 @@ function ModerationUsersList() {
       /> */}
       <div className='user-list__wrapper'>
         <table className='user-list__table'>
-          <thead className='user-list__table--header'>
-            <tr className='user-list__table--header--row'>
-              <th className='user-list__table--header--cell'>{t('user.email')}</th>
-              <th className='user-list__table--header--cell'>{t('user.name')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.role')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.verified')}</th>
-              <th className='user-list__table--header--cell'>{t('moderation.actions')}</th>
+          <thead className='user-list__table-header'>
+            <tr className='user-list__table-header-row'>
```

Comment:
is there any rule when to use __ or - or -- ? why is it so different in so many places in the code?